src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java

```diff
@@ -227,56 +227,84 @@ public void completeTask(String taskId,
         if (query == null) {
             return emptyList();
         }
+        try {
+            //1. Perform the search
```

Comment:
I'd prefer a method with a good name rather than the comment :)

Reply:
tbh, its just a one line call I don't think to keep it in a separate method.